main.c

```diff
@@ -0,0 +1,121 @@
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <ctype.h>

```

Comment:
Be considerate about the #includes. Let's not use includes if not absolutely necessary.